### PR TITLE
Update it.m3u logos

### PR DESCRIPTION
**Pull request by YacineBoussoufa**

### CODE REVIEW

File: channels/it.m3u [hunk 1]
```diff
@@ -240,15 +240,15 @@ https://5cbd3bc28341f.streamlock.net:444/telesud/_definst_/live/playlist.m3u8
 https://59d7d6f47d7fc.streamlock.net/rs2/rs2/playlist.m3u8
 #EXTINF:-1 tvg-id="TeleVideoAgrigento.it" tvg-name="TeleVideo Agrigento" tvg-country="IT" tvg-language="Italian" tvg-logo="https://i.imgur.com/crMQmn4.png" group-title="",TeleVideo Agrigento (480p)
 https://59d7d6f47d7fc.streamlock.net/tva/tva/playlist.m3u8
-#EXTINF:-1 tvg-id="TevereTV.it" tvg-name="Tevere TV" tvg-country="IT" tvg-language="Italian" tvg-logo="http://www.teveretv.it/Images/Icone/logo.png" group-title="",Tevere TV (576p)
```
Comment (freearhey) [inline channels/it.m3u:243]: A working link to the logo.

File: channels/it.m3u [hunk 2]
```diff
@@ -174,19 +174,19 @@ https://mediapolisevent.rai.it/relinker/relinkerServlet.htm?cont=0000001
...
+#EXTINF:-1 tvg-id="RetesoleLazio.it" tvg-name="Retesole Lazio" tvg-country="IT" tvg-language="Italian" tvg-logo="" group-title="",Retesole Lazio (240p)
 http://5c389faa13be3.streamlock.net:1935/8058/8058/playlist.m3u8
-#EXTINF:-1 tvg-id="Reteveneta.it" tvg-name="Reteveneta" tvg-country="IT" tvg-language="Italian" tvg-logo="https://i.imgur.com/kEje57b.png" group-title="",Reteveneta (480p)
+#EXTINF:-1 tvg-id="Reteveneta.it" tvg-name="Reteveneta" tvg-country="IT" tvg-language="Italian" tvg-logo="https://upload.wikimedia.org/wikipedia/it/d/df/Logo_Rete_Veneta.png" group-title="",Reteveneta (480p)
 https://59d7d6f47d7fc.streamlock.net/reteveneta/reteveneta/playlist.m3u8
 #EXTINF:-1 tvg-id="RMKTVSciacca.it" tvg-name="RMK TV Sciacca" tvg-country="IT" tvg-language="Italian" tvg-logo="https://i.imgur.com/Pk5flAT.png" group-title="",RMK TV Sciacca (720p)
 http://vod1.kronopress.com:1935/tmk_live/5123-CA5C-9EBE-428A/playlist.m3u8
-#EXTINF:-1 tvg-id="RTCQuartaRete.it" tvg-name="RTC Quarta Rete" tvg-country="IT" tvg-language="Italian" tvg-logo="https://i.imgur.com/SAuAcpv.png" group-title="",RTC Quarta Rete (576p)
```
Comment (freearhey) [inline channels/it.m3u:189]: A working link to the logo.

File: channels/it.m3u [hunk 3]
```diff
@@ -174,19 +174,19 @@ https://mediapolisevent.rai.it/relinker/relinkerServlet.htm?cont=0000001
...
+#EXTINF:-1 tvg-id="RDSSocialTV.it" tvg-name="RDS Social TV" tvg-country="IT" tvg-language="Italian" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/7/73/RDS-Logo.png" group-title="",RDS Social TV
 https://stream.rdstv.radio/out/v1/ec85f72b87f04555aa41d616d5be41dc/index_7.m3u8
 #EXTINF:-1 tvg-id="ReggioTV.it" tvg-name="ReggioTV" tvg-country="IT" tvg-language="Italian" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/60/Reggio_TV_logo.png" group-title="",ReggioTV (480p)
 https://59d7d6f47d7fc.streamlock.net/reggiotv/reggiotv/playlist.m3u8
-#EXTINF:-1 tvg-id="Rete4.it" tvg-name="Rete 4" tvg-country="IT" tvg-language="Italian" tvg-logo="" group-title="",Rete 4
+#EXTINF:-1 tvg-id="Rete4.it" tvg-name="Rete 4" tvg-country="IT" tvg-language="Italian" tvg-logo="https://upload.wikimedia.org/wikipedia/it/c/c0/Rete_4_-_Logo_2018.svg" group-title="",Rete 4
 https://live3-radio-mediaset-it.akamaized.net/Content/hls_h0_clr_vos/live/channel(r4)/index.m3u8
-#EXTINF:-1 tvg-id="RetesoleLazio.it" tvg-name="Retesole Lazio" tvg-country="IT" tvg-language="Italian" tvg-logo="https://i.imgur.com/vvVgZN2.png" group-title="",Retesole Lazio (240p)
```
Comment (freearhey) [inline channels/it.m3u:183]: A working link to the logo.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please explain why you decided to remove the marked logos?

**Comment by YacineBoussoufa** [comment]
> Please explain why you decided to remove the marked logos?

Sorry, while i was searching better logos i deleted them. I fixed.

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
Thanks!

**Pull request merged**